src/shared/components/common/markdown-textarea.tsx

```diff
@@ -331,6 +337,24 @@ export class MarkdownTextArea extends Component<
     );
   }
 
+  getCharacterCounter(contentLength: number, maxLength: number) {
+    return (
+      <>
+        <div className="flex-grow-1"></div>
```

Comment:
Probably don’t need an empty flex-grow. Try `justify-content-end` instead.

Reply:
I added `ms-auto`. It behaves slightly differently, if the character count is on its own line. Before it would show at the start of the line then, now it shows at the end. Though I think it looks better with `ms-auto`